src/particles.jl

```diff
@@ -0,0 +1,5 @@
+include("particles/particle_types.jl")
+include("particles/particle_direction.jl")
+include("particles/particle_spin_pol.jl")
+include("particles/particle_spinors.jl")
+include("particles/particle_states.jl")
```

Comment:
Suggestion: This should be moved directly to `QEDbase.jl` instead of piping it through a file.